.python_package/setup.cfg

```diff
@@ -0,0 +1,44 @@
+[metadata]
+name = gridtools-cpp
+author = ETH Zurich
+author_email = gridtools@cscs.ch
+license_files = LICENSE
```

Comment:
I  would like to add a comment here to document the fact that the License file is being copied in the building process... (but `configparser` would delete the comment later :sweat:   )